shared/elements/gaia_grid/js/items/group.js

```diff
@@ -183,10 +188,15 @@
 
           var itemVisible = (i - (index - nApps)) < COLLAPSED_GROUP_SIZE;
           if (!itemVisible) {
-            item.setCoordinates(x - width, y);
+            var xAxis = isRTL ? x + width : x - width;
```

Comment:
nit, call this xCoord or transformedX, or even just move the statement into the setCoordinates call and get rid of the variable altogether.